src/app/app.component.ts

```diff
@@ -1,10 +1,20 @@
 import { Component } from '@angular/core';
+import { first } from 'rxjs';
+import { ContactsService } from './contacts.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+  providers: [ContactsService]
 })
+
 export class AppComponent {
   title = 'angular-address-book';
+  
+  constructor(private contactsService: ContactsService){
```

Comment:
It looks like you're not using the `ContactsService` inside the main `App` component so we can remove this.